src/interface/desktop/loading-animation.js

```diff
@@ -0,0 +1,129 @@
+var $wrap = document.getElementById('loading-animation'),
```

Comment:
Would be fabulous to include some screenshots! 🌈

Reply:
Done!